sre/roles/tools/tasks/reinit_clickhouse.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Empty-drain-truncate relevant ClickHouse tables
```

Comment:
To reinforce this against name changes, think there is a way we can grab all of the existing names of the tables and the use that result to make this loop?